video_prediction/lstm_ops.py

```diff
@@ -99,7 +97,9 @@ def basic_conv_lstm_cell(inputs,
                             scope='Gates')
 
     # i = input_gate, j = new_input, f = forget_gate, o = output_gate
-    i, j, f, o = tf.split(axis=3, num_or_size_splits=4, value=i_j_f_o)
+
+    i, j, f, o = tf.split(i_j_f_o, 4,3 )
```

Comment:
The order of named arguments doesn't matter, as long as the name is consistent with the argument passed in. So, these two are equivalent.